web/assets/select.js

```diff
@@ -34,8 +34,18 @@ function enhanceElement(element, source) {
 }
 
 function fetchUser(prefix) {
+  let controller = { abort: () => {} };
+  const fetchOptions = {};
+
   return (query, callback) => {
-    fetch(`${prefix}/search-users?q=${encodeURIComponent(query)}`)
+    controller.abort();
+
+    if ("AbortController" in window) {
+      controller = new AbortController();
+      fetchOptions.signal = controller.signal;
+    }
+
+    fetch(`${prefix}/search-users?q=${encodeURIComponent(query)}`, fetchOptions)
```

Comment:
Could a helper function be extracted here - looks like the functions are the same aside from the `search-users` the url

Reply:
Good idea, I've extracted them out now 👍🏽